include/field_map_change.h

```diff
@@ -1,19 +1,19 @@
-#ifndef POKEPLATINUM_UNK_020530C8_H
-#define POKEPLATINUM_UNK_020530C8_H
+#ifndef POKEPLATINUM_FIELD_MAP_CHANGE_H
+#define POKEPLATINUM_FIELD_MAP_CHANGE_H
 
 #include "field/field_system_decl.h"
 #include "struct_defs/struct_02049FA8.h"
 #include "struct_decls/struct_020508D4_decl.h"
 #include "functypes/funcptr_02050904.h"
 
 void sub_020531A0(FieldSystem * fieldSystem);
-void sub_020531C0(FieldSystem * fieldSystem, BOOL param1);
-void sub_020532A8(FieldSystem * fieldSystem, BOOL param1);
+void FieldSystem_MapChange_UpdateGameData(FieldSystem * fieldSystem, BOOL param1);
+void FieldSystem_MapChange_UpdateGameDataDistortionWorld(FieldSystem * fieldSystem, BOOL param1);
```

Comment:
I'd like to propose that these be prefixed by module, e.g.,

```suggestion
void FieldMapChange_UpdateGameData(FieldSystem * fieldSystem, BOOL param1);
void FieldMapChange_UpdateGameDataDistortionWorld(FieldSystem * fieldSystem, BOOL param1);
```

Thoughts? I can foresee some lack of exposure here in typing `FieldSystem_` and having these functions omitted from autocomplete suggestions, but I think `FieldSystem_` is going to be so large that it will largely be irrelevant.

Reply:
I think that you're right, and the shorter name will be much more readable